Library/Homebrew/extend/os/linux/keg_relocate.rb

```diff
@@ -51,7 +51,7 @@ def change_rpath(file, old_prefix, new_prefix)
       cmd << "--set-interpreter" << new_interpreter if old_interpreter != new_interpreter
     end
 
-    return if old_rpath == new_rpath && old_interpreter == new_interpreter
+    return if old_rpath == new_rpath && old_interpreter.present? && old_interpreter == new_interpreter
```

Comment:
If `old_intepreter` is `nil`, the `patchelf` command will always run, even when `RPATH` are identical, which is not the desired behaviour.

Reply:
i've modified it to 
` return if old_rpath == new_rpath && (old_interpreter.blank? || old_interpreter == new_interpreter)`


but somewhere inside i feel, 
`return if old_interpreter.blank?  ||  (old_rpath == new_rpath && old_interpreter == new_interpreter)`
reads better.

Reply:
Please revert this line. The above suggestion handles this case.